### PR TITLE
Photoz DIR

### PR DESCRIPTION
**Pull request by anicola**
This PR implements the changes to config files needed to run DIR photometric redshift distribution estimation in TXPipe.

### CODE REVIEW

File: hsc/configs/hscconfig.yml [hunk 1]
```diff
@@ -72,6 +72,18 @@ TXTruthLensSelector:
...
 
+TXCosmosWeight:
+    mag_i_cut: 24.5
+    n_neighbors: 10
+
+TXPhotozDIR:
+    pz_code: 'ephor_ab'
+    pz_mark: 'best'
+    source_zbin_edges: [0.3, 0.6, 0.9, 1.2, 1.5]
+    lens_zbin_edges: [0.15,0.5,0.75,1.0,1.5]
```
Comment (fjaviersanchez) [inline hsc/configs/hscconfig.yml:83]: This means that I didn't update this config properly myself ¬¬. Could you please set the same source and lens bins?

**Review by fjaviersanchez — APPROVED** [review]
Looks good! thanks

**Comment by anicola** [comment]
@fjaviersanchez Thank you so much! I've implemented all the changes.

**Review by fjaviersanchez — APPROVED** [review]
Looks good! Thanks :)

**Pull request merged**